src/Network/Packets/ServerToClient/ServerToClientPackets.xml

```diff
@@ -5250,6 +5250,75 @@
         </Field>
       </Fields>
     </Packet>
+    <Packet>
+      <HeaderType>C1HeaderWithSubCode</HeaderType>
+      <Code>BF</Code>
+      <SubCode>51</SubCode>
+      <Name>MuBotUseResponse</Name>
+      <Length>16</Length>
+      <Direction>ServerToClient</Direction>
+      <SentWhen>The server validate (and toggle status) if user can use the bot</SentWhen>
+      <CausedReaction>The client toggle the mu bot status</CausedReaction>
+      <Fields>
+        <Field>
+          <Index>4</Index>
+          <Type>Byte</Type>
+          <Name>IsZen</Name>
+        </Field>
+        <Field>
+          <Index>5</Index>
+          <Type>Byte</Type>
+          <Name>Unknown1</Name>
+        </Field>
+        <Field>
+          <Index>6</Index>
+          <Type>Byte</Type>
+          <Name>Unknown2</Name>
+        </Field>
+        <Field>
+          <Index>7</Index>
+          <Type>Byte</Type>
+          <Name>Unknown3</Name>
+        </Field>
```

Comment:
can be removed - it's padding because of struct field alignment

Reply:
done